website/content/en/docs/troubleshooting.md

```diff
@@ -325,6 +325,17 @@ then the following solution(s) may resolve your issue.
 ...
 ```
 
+### Karpenter incorrectly computes available memory for a node
+
+Karpenter uses the results from `ec2:DescribeInstanceTypes` to determine the memory available on a node launched with an instance type.
+However, in addition to the raw memory available on an instance, there are several factors that affect the memory that will be available to pods when scheduling.
+Some of these are driven by user configuration, for example the kubelet configuration options `kubeReserved` and `systemReserved`.
+But one factor is implicit, and varies by combination of instance type and AMI.
+Although Karpenter can't compute this value directly, it models it as `VM_MEMORY_OVERHEAD_PERCENT`.
+By default this value is set to `7.5%`, a value that we have determined closely matches reality for most instance types without overestimating.
+This value can be tuned to bring Karpenter's estimate closer to reality for the instance types in your cluster, but this should be done with caution.
+Changing this value such that Karpetner overestimates the memory available on a node can result in an instance launch loop as Karpenter provisions undersized nodes for your workloads.
```

Comment:
Should we mention that the NodeClaim is a good way to check if these values are matching what you would expect?

Reply:
Should we mention that validating the presence of ConsistentStateFound status condition is a good way to ensure that you catch issues like this early? Maybe even reference the metric that you can use to monitor this? `operator_status_condition...`